src/analyzer/protocol/netbios/NetbiosSSN.cc

```diff
@@ -10,14 +10,15 @@
 #include "zeek/analyzer/protocol/netbios/events.bif.h"
 #include "zeek/session/Manager.h"
 
-constexpr double netbios_ssn_session_timeout = 15.0;
-
-#define MAKE_INT16(dest, src)                                                                                          \
-    (dest) = *(src);                                                                                                   \
-    (dest) <<= 8;                                                                                                      \
-    (src)++;                                                                                                           \
-    (dest) |= *(src);                                                                                                  \
-    (src)++;
+static constexpr double netbios_ssn_session_timeout = 15.0;
```

Comment:
Would you be up for making this a script-level variable in a `const.bif` somewhere and accessing it with `BifConst::` ? I don't think that's a lot of work and might add a bit of value to someone out there ;-)

Reply:
I'll make a note to do it separately to this PR.